<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Analysis"/>
    <addaction name="actionSave_Analysis"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_File_to_Analyze"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Analysis">
   <property name="text">
    <string>Open Analysis</string>
   </property>
  </action>
  <action name="actionOpen_File_to_Analyze">
   <property name="text">
    <string>Add File to Analysis</string>
   </property>
  </action>
  <action name="actionSave_Analysis">
   <property name="text">
    <string>Save Analysis</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
